android/src/main/java/org/ligi/passandroid/ui/quirk_fix/USAirwaysLoadActivity.kt

```diff
@@ -13,12 +15,38 @@ class USAirwaysLoadActivity : PassAndroidActivity() {
         super.onCreate(savedInstanceState)
 
         val data = intent.data
+        if (data == null){
```

Comment:
seems the action following is the same code as in URLRewriteActivity.java - should be made DRY - very WET now